help/help.adoc

```diff
@@ -2,874 +2,576 @@
 Plugin Version @PLUGIN_VERSION@
 Revised on April 14, 2022
 
-== Integrated version
+== Supported versions
 
 This plugin was developed and tested against OpenStack IceHouse.
 
-The minimum {CD} supported version is ElectricCommander 3.6.
-
-IMPORTANT: For all parameter descriptions in this document, the required parameters are shown in [.required]#bold italics#.
+The minimum {PRODUCT} supported version is ElectricCommander 3.6.
```

Comment:
In release notes(1.4.0), minimum supported version is 10.3. So please check it.https://github.com/electric-cloud/EC-OpenStack/pull/93/files#diff-f17cd74813ef49ee0f14269b7267209fc94c9a5d918b963220b613748f776188R496

Reply:
Thanks, @cmarunachalam! It looks like the 1.4.0 version of the plugin has not been released yet: https://docs.cloudbees.com/plugins/cd/ec-openstack, so I was planning to remove the 1.4.0 release notes from the documentation PR before merging it: https://github.com/cloudbees/docsite-cloudbees-common-sda/pull/659/commits/6d4585f2116319f2030d5871eb0b6552e85890ec.

Should I still go ahead and update this to 10.3?

Reply:
(Or, if we plan to release v1.4.0 very soon, I can make the necessary updates to this PR, not merge the current documentation PR, and the documentation will be updated with my changes when it is released.)

Reply:
I have no idea about the v1.4.0 release. If you are going to remove the v1.4.0 release notes then no issue. You don't need to update the version to 10.3.